### PR TITLE
style: JSX(only UI) of Header Component

### PR DESCRIPTION
**Pull request by tushartiwari7**
## Created Header of E-Commerce Application
- ONLY JSX, no functionalities right now.

### Major Files Added
- [Header.jsx](https://github.com/tushartiwari7/furncastle/compare/header?expand=1#diff-d01b8b59c41207ee34488c2bb96754ecc7505ba22428ba5009e867e59119105a)
- [Header.css](https://github.com/tushartiwari7/furncastle/compare/header?expand=1#diff-d01b8b59c41207ee34488c2bb96754ecc7505ba22428ba5009e867e59119105a)

localhost_3000_

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    </div>
+    <div className="flex flex-center">
+      <i className="bi bi-search search-icon mx-xs"></i>
+      <a  href="/pages/login.html" className="btn btn-outline-primary btn-login px-sm py-xs mx-xs fs-s">
+        Login
+      </a>
+      <a href="/pages/wishlist.html" className="list white">
+        <i className="bi bi-heart mx-xs"></i>
+      </a>
+      <a href="/pages/cart.html" className="list white">
```
Comment (Bhakti199) [inline src/components/Header/Header.jsx:30]: attach file after adding it.

**Review by Bhakti199 — APPROVED** [review]
this PR is good to merge

File: src/components/Header/Header.jsx (hunk 1 above)
Comment (tushartiwari7) [reply src/components/Header/Header.jsx:30]: Sure! will do it.

File: src/components/Header/Header.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  background: white; 
+}
+.btn-login:hover,.hamburger:hover {
+  /* primary color by default */
+  border-color: white; 
+}
+
+.input {
+  background: white;
+}
```
Comment (ozairkhan2810) [inline src/components/Header/Header.css:13]: can add input class with 2nd line because same properties
also remove comments if not necessary

**Review by ozairkhan2810 — APPROVED** [review]
just one suggestion, rest looks good

File: src/components/Header/Header.css (hunk 2 above)
Comment (tushartiwari7) [reply src/components/Header/Header.css:13]: Sure will do it in next PR!

**Pull request merged**